web/src/app/modules/sugarloaf/components/smart/navigation/navigation.component.html

```diff
@@ -1,58 +1,106 @@
+        <ng-template [ngIf]="section.module">
+          <div class="nav-divider"></div>
+        </ng-template>
 
-    <clr-tree-node
-      *ngFor="let category of section.children; trackBy: identifyNavigationItem"
-      [clrExpanded]="true"
-      [clrLoading]="category.isLoading"
-    >
-      <a
-        [routerLinkActiveOptions]="{ exact: true }"
-        [routerLink]="category.path | formatpath"
-        class="clr-treenode-link category"
-        routerLinkActive="active"
-      >
-        <span>
-          <clr-icon
-            [attr.shape]="itemIcon(category) | default: 'folder'"
-          ></clr-icon>
-          {{ category.title }}
-        </span>
-      </a>
+        <ng-container *ngIf="section.children?.length > 0; else noChildCollapsed">
+          <a clrVerticalNavLink
+             (click)="openPopup(i)"
+             [ngClass]= "{'item-selected' : i == this.lastSelection}"
+             (mouseover)="flyoutIndex= i; lastSelection=i">
+            <clr-icon [attr.shape]="section.iconName | default:'home'" clrVerticalNavIcon></clr-icon>
+          </a>
 
-      <clr-tree-node
-        *ngFor="let entry of category.children; trackBy: identifyNavigationItem"
-        [clrLoading]="entry.isLoading"
-      >
-        <a
-          [routerLink]="entry.path | formatpath"
-          class="clr-treenode-link item"
-          routerLinkActive="active"
-        >
-          <clr-icon
-            [attr.shape]="itemIcon(entry) | default: 'folder'"
-          ></clr-icon>
-          <span>
-            {{ entry.title }}
-          </span>
-        </a>
-      </clr-tree-node>
-    </clr-tree-node>
-  </clr-tree-node>
-</clr-tree>
+          <ng-container *ngIf="shouldExpand(i)">
+            <div class="flyout nav-group-children">
+              <span class="flyout-header">{{ section.title }}</span>
+              <clr-vertical-nav-group-children class="flyout-group">
+                <a
+                  *ngFor="let category of section.children; trackBy: identifyNavigationItem"
+                  [routerLink]="formatPath(category.path)"
+                  [routerLinkActiveOptions]="{exact: true}"
+                  routerLinkActive="active"
+                  class="flyout-item nav-link"
+                  (click)="setNavState(false, i)"
+                >
+                {{ category.title }}
+                </a>
+              </clr-vertical-nav-group-children>
+            </div>
+          </ng-container>
+        </ng-container>
+
+        <ng-template #noChildCollapsed>
+              <span class="tooltip-override tooltip tooltip-md">
+                <a clrVerticalNavLink
+                   [ngClass]= "{'item-selected' : i == this.lastSelection}"
+                   [routerLinkActiveOptions]="{exact: true}"
+                   [routerLink]="formatPath(section.path)"
+                   routerLinkActive="active"
+                   (mouseover)="flyoutIndex= -1; lastSelection= i"
```

Comment:
Upon moving the mouse away from the collapsed nav, the item should not remain highlighted